stablehlo/tests/interpret/sort.mlir

```diff
@@ -15,3 +15,16 @@ func.func @sort_stable() {
   check.expect_eq_const %result1, dense<[[1, 2, 1], [3, 2, 3]]> : tensor<2x3xi64>
   func.return
 }
+
+// -----
+
+func.func public @sort() {
```

Comment:
can we rename to sort_issue_x as precedent? 
https://github.com/openxla/stablehlo/blob/eba821aa1c54a21d70331d7926dfc8b929f988f3/stablehlo/tests/interpret/all_gather.mlir#L52

Reply:
done.